benchmarks/utils/update_artifacts.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+import os
+import shutil
+
+BENCHMARK_REPORT_PATH = "/tmp/ts_benchmark"
+BENCHMARK_ARTIFACTS_PATH = "/tmp/ts_artifacts"
+BENCHMARK_REPORT_FILE = "ab_report.csv"
+WINDOW_LEN = 8
+WINDOW_START = 0
+
+# Copy BENCHMARK_REPORT_FILE to artifacts
+def copy_benchmark_reports(input, output):
+
+    for dir in os.listdir(input):
+        if os.path.isdir(os.path.join(input, dir)):
+            new_dir = os.path.join(output, dir)
+            os.makedirs(new_dir, exist_ok=True)
+            shutil.copy(os.path.join(input, dir, BENCHMARK_REPORT_FILE), new_dir)
+
+
+# Create artifacts for a period of rolling WINDOW_LEN-1 reports
+def update_artifacts(args):
```

Comment:
nit: Instead of an args object, opt for either explicitly showing what's being passed in or for a dataclass. Also input is a reserved keyword in python so might be best to pick something like input_dir

Reply:
Done